### DISCUSSION TITLE
应该怎么输入化学中的pH?

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#110] [Q&A]
你好，
在化学中，常说pH，pH这个词不是直接输入的吧？直接输入是两个字母，那么我应该怎么输入pH这个词？
谢谢

**Comment by tanukihee** [comment]
前提就错了，就是直接输入的

**Reply by earthmoon** [reply]
> 前提就错了，就是直接输入的

这样呀，好吧。

**Comment by tanukihee** [comment]
> 直接输入是两个字母

难以理解，所有的西文单词不都是由字母组成的？他又没有特殊的意义（既不是单位，又不是化学符号），为什么要特殊对待

**Reply by earthmoon** [reply]
因为，比如一个模板的英文字体默认是斜体，那么我们在这个模板中提到pH应该是斜体，但是pH正常的是正体。pH也不可以加粗的吧，但是这些都会受到默认的环境影响。如果它像一个单位一样是有固定的表达，就可以避免这些问题。

**Reply by tanukihee** [reply]
依我理解，pH 只是一个单词的缩写，没有特殊地位，该斜体斜体，该加粗加粗。并不像符号、单位那样具有特殊意义。

**Reply by earthmoon** [reply]
> 依我理解，pH 只是一个单词的缩写，没有特殊地位，该斜体斜体，该加粗加粗。并不像符号、单位那样具有特殊意义。

有道理！它的本质只是缩写，没有理由区别对待。
